.github/workflows/packaging-test-pipelines.yml

```diff
@@ -0,0 +1,56 @@
+name: Packaging Tests
```

Comment:
We usually name tests on Citus repo as such: `test_check-build`

Reply:
changed to test_packaging

Reply:
Rather than "packaging", "build" sounds like a more appropriate naming to me; given that we're not building any kind of packages but `make`ing Citus on different distros.

Reply:
changed to tests_build_in_packaging_images. I need everyone to be concious of this is about packaging since we are building in packaging images, not in vanilla docker images.

Reply:
yeah that makes more sense

Reply:
I see `Packaging Tests` here right now. I like the option `tests_build_in_packaging_images` better as Gurkan shared it in the comments above.

Reply:
Is it enough to have this in a job name only? Can we have a more descriptive workflow name as well?

Reply:
Changed the workflow name to Build tests in packaging images